travis_configure.sh

```diff
@@ -38,12 +38,10 @@ before_install()
             build
 
         # jfrog dependency
-        if [[ "$MAKE_TARGET" =~ "deb" ]]; then
-            if [[ -n "$TRAVIS_TAG" && $TRAVIS_REPO_SLUG == "sociomantic-tsunami/libdrizzle-redux" ]]; then
-                curl -XGET -L -k 'https://api.bintray.com/content/jfrog/jfrog-cli-go/$latest/jfrog-cli-linux-amd64/jfrog?bt_package=jfrog-cli-linux-amd64' > /tmp/jfrog ;
-                chmod a+x /tmp/jfrog ;
-                sudo cp /tmp/jfrog /usr/local/bin/jfrog ;
-            fi
+        if [[ -n "$TRAVIS_TAG" ]]; then
+            curl -XGET -L -k 'https://api.bintray.com/content/jfrog/jfrog-cli-go/$latest/jfrog-cli-linux-amd64/jfrog?bt_package=jfrog-cli-linux-amd64' > /tmp/jfrog ;
+            chmod a+x /tmp/jfrog ;
+            sudo cp /tmp/jfrog /usr/local/bin/jfrog ;
```

Comment:
Are these 2 changes really related, couldn't you change the conditions without touching this file?

Reply:
No, need to remove the **TRAVIS_REPO_SLUG** since `jfrog` wouldn't be available in a tagged build from a fork.